src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { config } from "dotenv";
+import { run } from "./cronjob";
+import { CronJobOptions } from "./types";
+config();
+
+const cronJobOptions: CronJobOptions = {
+  ical: {
+    url: process.env.ICAL_URL,
```

Comment:
Out of curiosity, did we drop support for multiple urls?

Reply:
This can definitely be improved (as it is not explicitly exposed), but we did not lose the support / functionality as we still enable it here:
https://github.com/jkrnak/ical-slack-vacations/pull/13/files#diff-177a3d9b6fe25933cc93ec453bbff463R11
```javascript
// src/ical/index.ts
// line 11
export async function fetchInfo(
  url: string,
  { debug = false, categoryKey = "categories" }: IcalClientOptions
): Promise<CalendarResult> {
  const icalUrls = url.split(",");
```

Here we just extracted the options as `url` and then we still try to split it by `,` if exists